File: lib/src/models/following.dart
class Following {
  final int id;
  final int follower;
  final int followed;

  Following({
    required this.id,
    required this.follower,
    required this.followed,
  });

  factory Following.fromJson(Map<String, dynamic> json) {
    return Following(
      id: json['id'],
      follower: json['follower'],
      followed: json['followed'],
    );
  }
}

File: android/app/src/main/kotlin/com/example/yogi_application/model/Following.kt
package com.example.yogi_application.model

import kotlinx.serialization.Serializable
import kotlinx.serialization.encodeToString
import kotlinx.serialization.json.Json

@Serializable
data class Following(
    val id: Int,
    val follower: Int,
    val followed: Int
) {
    companion object {
        fun fromJson(source: String): Following {
            return Json.decodeFromString(source)
        }
    }

    fun toJson(): String {
        return Json.encodeToString(this)
    }
}